components/common/party/Cusor.vue

```diff
@@ -39,5 +39,32 @@ const props = defineProps<{
   cursorDetails: CursorDetails
 }>()
 
+const colors = [
```

Comment:
still getting duplicate colors. green and gray

<img width="348" alt="Screenshot 2024-03-05 at 16 43 19" src="https://github.com/kodadot/nft-gallery/assets/734428/4add1ca1-9914-4447-8254-6051a1f6517e">

Reply:
Moving the `colors` var to `Cursor.vue` doesn't fix the duplicate issue. You can do this instead:

1. get colour by `index` number. add index in this line `v-for="connection in connections"`
2. if you want to randomize the colours. shuffle the array instead of randomize the index number. `shuffleArray([...colors])` instead of `Math.floor(Math.random() * colors.length)`
3. because tailwind colours are limited only to `n` length, randomize hex colours for cursor `> n length`